subgraphs/erc721/schema.graphql

```diff
@@ -0,0 +1,199 @@
+###
+# Subgraph Schema: ERC721 Smart Contracts and Tokens
+# Version: 1.0.0
+
+#############################
+####### Token Metadata ######
+#############################
+
+type Token @entity {
+  " address of the token, { Contract Address }-{ Token ID }  "
+  id: ID!
+
+  " Smart contract for the token "
+  contract: Contract!
+
+  " ID of the token from the smart contract "
+  tokenID: BigInt!
+
+  " URI of the token "
+  tokenURI: String
+
+  " The owner for the token "
+  owner: Account!
+
+  " timestamp of the token's mint "
+  mintTime: BigInt!
+}
+
+#############################################
+####### ERC721 Smart Contract Metadata ######
+#############################################
+
+type Contract @entity {
```

Comment:
Can you rename this to `Collection`?

Reply:
Yes. Modified at https://github.com/messari/subgraphs/pull/689/commits/94a0e318f6371aaff194d76f3e621577de2d5f69